src/libs/API/parameters/index.ts

```diff
@@ -165,6 +165,7 @@ export type {default as CreateWorkspaceCategoriesParams} from './CreateWorkspace
 export type {default as RenameWorkspaceCategoriesParams} from './RenameWorkspaceCategoriesParams';
 export type {default as SetWorkspaceRequiresCategoryParams} from './SetWorkspaceRequiresCategoryParams';
 export type {default as DeleteWorkspaceCategoriesParams} from './DeleteWorkspaceCategoriesParams';
```

Comment:
NAB - damn i wish we could rename all of these to "Policy" for consistency...... But not really appropriate in this PR lol